docs/advanced/bootloader.md

```diff
@@ -94,3 +98,18 @@ It produces the following ROM layout:
 ``` 
 
 For an example showing how to create an application that uses a bootloader, see the [mbed OS bootloader example](https://github.com/armmbed/mbed-os-example-bootloader-blinky) repository.
+
+## Unmanaged bootloader
+
+You want to have an unmanaged bootloader when your bootloaders requirements conflict with the requirements of the managed bootloader. You need an unmanaged bootloader when your bootloader does not come before your application in ROM or your application does not start immediately after your bootloader. Unlinke a managed bootloader, an unmananged bootloader does not automatically merge the bootloader image with the application image after building the application. We expect a user of an unmanaged bootloader build to construct there own set of scripts built atop the `mbed compile` primitive to perform bootloader and application merging.
+
+An unmanaged bootloader build is a method for controlling the link location of a program within mbed OS. There are two configuration options available for changing the link location: `target.mbed_app_start` and `target.mbed_app_size`. 
```

Comment:
Query: Should this appear earlier, before we first mention "unmanaged bootloaders"?